lib/bench.js

```diff
@@ -11,32 +12,36 @@ const doBench = async (opts, handler) => {
     spinner.color = 'magenta'
     spinner.text = `Warming ${handler}`
     await fire(opts, handler, false)
-  } catch (e) {
-    console.log(e)
+  } catch (error) {
+    return console.log(error)
   } finally {
     spinner.color = 'yellow'
     spinner.text = `Working ${handler}`
   }
+
   try {
     await fire(opts, handler, true)
     forked.kill('SIGINT')
     spinner.text = `Results saved for ${handler}`
     spinner.succeed()
     return true
-  } catch (e) {
-    console.log(e)
+  } catch (error) {
+    return console.log(error)
   }
 }
 
-const start = async (opts, list, inc = 0) => {
```

Comment:
It looks more compilcated then previous version I think.

Reply:
@cagataycali -How does it look more complicated? -You should try to avoid re-assigning variables and that's exactly why I made this change. Also, IMO, `index` reflects better what is going on, `inc` does not so much.

Reply:
You're right